scavenger-old-agent-java/build.gradle.kts

```diff
@@ -36,6 +36,9 @@ dependencies {
 java {
     withJavadocJar()
     withSourcesJar()
+    toolchain {
+        languageVersion.set(JavaLanguageVersion.of(8))
```

Comment:
What do you think about discontinuing Java 7 support from now on?

Reply:
I think that's good.
If we keep supporting java 7, we'll have to give up more becuz we have to consider minimum java version compatibility.